File: grind-75/array/twoSum.py
from typing import List


class Solution:
    @classmethod
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        store = {}
        for i, num in enumerate(nums):
            if num in store:
                return [store[num], i]
            store[target - num] = i

File: projects/grind-75/grind-75-ts/array/two_sum.ts
function twoSum(nums: number[], target: number): number[] {
  const store = {};
  for (let [i, num] of nums.entries()) {
    if (num in store) {
      return [store[num], i];
    }
    store[target - num] = i;
  }
  return [];
}
